### DISCUSSION TITLE
Ip display companion pi

### DISCUSSION THREAD

**Discussion by thostie2002** [bitfocus/companion#2683] [Q&A]
Hi, 

im using companion pi on my raspberry pi3. Now i want to show the host ip on an oled, did anyone did this? And can you tell me how to?

**Comment by kraves-oz** [comment]
The internal variables can display IP address on a button title. 

$(internal....

**Reply by thostie2002** [reply]
Nice, but how to add it to a 0.9" oled display thats connected to the gpio's
